### DISCUSSION TITLE
Code server reported an error when using it. Is this my browser's problem?

### DISCUSSION THREAD

**Discussion by leaf2006** [coder/code-server#2406] [Q&A]
- Web Browser:Chrome
- Local OS:Mac os big sur
- Remote OS:Mac os
- Remote Architecture:amd64
- `code-server --version`:3.7.4

In my use, this error often appears:
```
加载 Web 视图时出错: Service Workers are not enabled in browser. Webviews will not work.
```
Hope someone can help me to answer, this is my browser problem? How to solve it?

**Comment by code-asher** [comment]
You'll need to access code-server over HTTPS. Browsers only allow
service workers (and other things) when the context is secure.
